.github/workflows/ci-cd.yml

```diff
@@ -23,6 +23,10 @@ jobs:
       - uses: eskatos/gradle-command-action@v1
         with:
           arguments: build
+      - name: Assemble instrumentation tests APK
+        uses: eskatos/gradle-command-action@v1
+        with:
+          arguments: "app:assembleAndroidTest"
```

Comment:
Is this the right task to use? I've also seen `app:assembleDebugAndroidTest`.

Reply:
That might default to the `debug` flavor, but it's probably best to be make it explicit with `assembleDebugAndroidTest`. Also to ensure there's also a `app-debug.apk`, I usually run before `assembleDebug`.